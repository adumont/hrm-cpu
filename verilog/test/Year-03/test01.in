5 01
5 02
5 03
5 04
5 05
5 06
5 07
5 08
5 09
5 0A
