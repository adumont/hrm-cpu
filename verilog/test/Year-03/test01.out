42
55
47
